### ISSUE TITLE
apt key for deb9

### ISSUE THREAD

**Issue by niktheworker** [Oefenweb/ansible-percona-server#39]
Hi. 

I run task repository.yml and have error:

`
fatal: [debian-9]: FAILED! => {"changed": false, "cmd": "/usr/bin/apt-key adv --no-tty --keyserver keyserver.ubuntu.com --recv 9334A25F8507EFA5", "msg": "Error fetching key 9334A25F8507EFA5 from keyserver: keyserver.ubuntu.com", "rc": 2, "stderr": "Warning: apt-key output should not be parsed (stdout is not a terminal)\ngpg: failed to start the dirmngr '/usr/bin/dirmngr': No such file or directory\ngpg: connecting dirmngr at '/tmp/apt-key-gpghome.2cEO2eEG5w/S.dirmngr' failed: No such file or directory\ngpg: keyserver receive failed: No dirmngr\n", "stderr_lines": ["Warning: apt-key output should not be parsed (stdout is not a terminal)", "gpg: failed to start the dirmngr '/usr/bin/dirmngr': No such file or directory", "gpg: connecting dirmngr at '/tmp/apt-key-gpghome.2cEO2eEG5w/S.dirmngr' failed: No such file or directory", "gpg: keyserver receive failed: No dirmngr"], "stdout": "Executing: /tmp/apt-key-gpghome.2cEO2eEG5w/gpg.1.sh --no-tty --keyserver keyserver.ubuntu.com --recv 9334A25F8507EFA5\n", "stdout_lines": ["Executing: /tmp/apt-key-gpghome.2cEO2eEG5w/gpg.1.sh --no-tty --keyserver keyserver.ubuntu.com --recv 9334A25F8507EFA5"]}
`

i resolved this problem:
1) Add new key 
2) Add install dirmng

```
- name: repository | install | dependencies
  apt: pkg=dirmngr state=installed update_cache=true

- name: repository | add public key
  apt_key:
    id: 8507EFA5
    keyserver: keys.gnupg.net
    state: present
  tags:
    - percona-server-repository-public-key

```

Best regards

**Comment by tersmitten** [comment]
This fix should already be present. You might be hitting [this error](https://github.com/ansible/ansible/pull/57814). Are you using Ansible 2.8?

**Comment by tersmitten** [comment]
Also see https://github.com/Oefenweb/ansible-fail2ban/issues/56

**Comment by niktheworker** [comment]
I use ansible 2.8.1, but i don't understand why deprecated keys for APT repo related this.

**Comment by tersmitten** [comment]
On Debian 9 the `dirmngr` package needs to be installed

**Comment by tersmitten** [comment]
This is already present in this role, but seems to be skipped

**Comment by tersmitten** [comment]
See https://github.com/Oefenweb/ansible-percona-server/blob/master/tasks/repository.yml#L11

**Comment by niktheworker** [comment]
Yes, i see. I resolved this: 
**after:**  ``` when: ansible_distribution == 'Debian' and ansible_distribution_version is version('**9.0**', '>=') ```
**before:** ```  when: ansible_distribution == 'Debian' and ansible_distribution_version is version('**9**', '>=') ```

**Comment by tersmitten** [comment]
Can you make a pull request?

**Comment by niktheworker** [comment]
Yes, i make tomorrow

**Comment by tersmitten** [comment]
Thanks

**Comment by tersmitten** [comment]
Fixed in 9a4f9954e8d20ceeb2824004d6fef250b1f38f0e
